src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -778,7 +778,7 @@ void IgnRenderer::Render()
 
   if (ignition::gui::App())
   {
-    gui::events::Render event;
+    ignition::gui::events::Render event;
```

Comment:
there are other places in ign-gazebo3 that uses gui::events::Render. Do they need to be updated as well? e.g. https://github.com/gazebosim/gz-sim/blob/ign-gazebo3/src/gui/plugins/grid_config/GridConfig.cc#L90

Reply:
Nope, the bug lived in the sender. Anything listening to the render event doesn't need updated since both a `ignition::gui::events::Render` and `ignition::gazebo::gui::events::Render` are being sent: https://github.com/gazebosim/gz-sim/blob/ign-gazebo3/include/ignition/gazebo/gui/GuiEvents.hh

Since both events are being broadcasted, plugins listening to either render event will be caught now (in fortress, the `ignition::gazebo::gui::events::Render` is removed)